### PR TITLE
[For only Review] Added ValidateBlock API for checking committe/proposer/seal of block header (Draft)

### PR DESCRIPTION
**Pull request by KimKyungup**
## Proposed changes

- Describe your changes to communicate to the maintainers why we should accept this pull request.
- If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: consensus/istanbul/backend/api.go [hunk 1]
```diff
@@ -214,11 +217,113 @@ func (api *APIExtension) GetCommitteeSize(number *rpc.BlockNumber) (int, error)
...
+	ProposerFromBlock        common.Address      `json:"proposerFromBlock"`
+	IsValidProposer          bool                `json:"isValidProposer"`
+	Committee                common.AddressSlice `json:"committee"`
+	CommitteeSealedFromBlock common.AddressSlice `json:"committeeSealedFromBlock"`
+	CommitteeFromBlock       common.AddressSlice `json:"committeeFromBlock"`
+	IsValidCommittee         bool                `json:"isValidCommittee"`
+	IsValidSeal              bool                `json:"isValidSeal"`
+}
+
+func (api *APIExtension) ValidateConsensusInfo(block *types.Block) (ValidationResult, error) {
```
Comment (KimKyungup) [inline consensus/istanbul/backend/api.go:233]: This validate code can check below.
- proposer from block is same with calculated proposer
- committe from block is same with calculated committe
- All sealed committe is in the calculated committe

File: consensus/istanbul/backend/api.go [hunk 2]
```diff
@@ -321,15 +458,55 @@ func (api *APIExtension) makeRPCBlockOutput(b *types.Block,
...
+	r["Round"] = cInfo.round
 	r["originProposer"] = cInfo.originProposer
+	r["roundProposer"] = cInfo.roundProposer
+	r["roundCommitte"] = cInfo.roundCommitte
 	r["transactions"] = rpcTransactions
 
 	return r
 }
 
+func (api *APIExtension) ValidateBlock(number *rpc.BlockNumber) (interface{}, error) {
```
Comment (aidan-kwon) [inline consensus/istanbul/backend/api.go:474]: I think "block validation" usually means validating a block header including state root and its transactions. But, this method validates block extra data. How about changing its name to `ValidateBlockSeal`?

File: consensus/istanbul/backend/api.go [hunk 3]
```diff
@@ -321,15 +458,55 @@ func (api *APIExtension) makeRPCBlockOutput(b *types.Block,
...
 	}
 
-	r["committee"] = cInfo.committee
-	r["proposer"] = cInfo.proposer
-	r["round"] = cInfo.round
+	r["Committee"] = cInfo.committee
+	r["committeeFromExtra"] = cInfo.validatorsFromExtra
+	r["committeeSealFromExtra"] = cInfo.committeeFromExtraSeal
+	r["Proposer"] = cInfo.proposer
+	r["Round"] = cInfo.round
```
Comment (aidan-kwon) [inline consensus/istanbul/backend/api.go:465]: Some keys start with a capital letter.

**Comment by KimKyungup** [comment]
I will separate this PR.

**Pull request closed without merging**